scripts/plantuml.sh

```diff
@@ -0,0 +1 @@
+java -Djava.awt.headless=true -jar assets/plantuml.jar -tsvg -failfast2 "$@"
```

Comment:
a) Please consider the sonatype-lift remark
b) If you can have a wish, I would ask you to not push the jar-file but download it on demand aka. call of the script